Networking/Networking/Model/POSProduct.swift

```diff
@@ -35,6 +35,10 @@ public struct POSProduct: Codable, Equatable, GeneratedCopiable, GeneratedFakeab
         attributes.filter { $0.variation }
     }
 
+    public let manageStock: Bool
+    public let stockQuantity: Decimal?
+    public let stockStatusKey: String
```

Comment:
```suggestion
    public let stockStatusKey: String
```

Based [on documentation](https://woocommerce.github.io/woocommerce-rest-api-docs/#list-all-products):

> `stock_status`  Options: instock, outofstock and onbackorder.

I think we should just encode it it directly into an enum, it will be easier to work with it.